ftplugin/markdown.vim

```diff
@@ -42,6 +42,18 @@ if !exists("g:vmt_include_headings_before")
     let g:vmt_include_headings_before = 0
 endif
 
+if !exists("g:vmt_link")
+    let g:vmt_link = 1
+endif
+
+if !exists("g:vmt_min_level")
+    let g:vmt_min_level = 1
+endif
+
+if !exists("g:vmt_max_level")
+    let g:vmt_max_level = 3
```

Comment:
I think default value of `g:vmt_max_level` is 6 better, otherwise it will change the default behavior of the plugin.